loadtest/user/userentity/user.go

```diff
@@ -39,6 +40,8 @@ const (
 	AuthenticationTypeSAML       = "saml"
 )
 
+var stripIDsRE = regexp.MustCompile(`\b\w{26}\b`)
```

Comment:
This regex is making me uncomfortable. What about file URLs? And also variations in query params? It feels like it leaves out a lot of corner cases. Can we take a simplistic approach where we make the buckets more coarse by reducing the path further, but we ensure that it's a clean solution?

Reply:
> What about file URLs? And also variations in query params?

Can you give examples of these? I've been using this sort of regex when manually grepping in the logs several times and I've never found an edge case (or I have not realized I did, of course :joy:)

Reply:
@agnivade It would literally capture just the ids at this point. Query params would not be full words due to the `?` so they wouldn't get captured unless the value itself is 26 chars (e.g. `/api/v4/something?query=aaaaaaaaaaaaaaaaaaaaaaaaaa`). 

We could add the `/` matching (before and after the word) and replace the resulting captured group if you really wanted that extra check, but we are talking about a metrics label in a load-test tool, not sure how strict we need to be here.

Reply:
Ah nvm, `Path` doesn't contain query params. So all good there. And I checked the file URLs, they also have an id so we should be fine for now.